packages/composite-checkout/src/components/checkout-steps.js

```diff
@@ -373,7 +373,7 @@ const MainContentUI = styled.div`
 	background: ${props => props.theme.colors.surface};
 	width: 100%;
 	box-sizing: border-box;
-	margin-bottom: ${props => ( props.isLastStepActive ? '89px' : 0 )};
+	margin-bottom: ${props => ( props.isLastStepActive ? '100px' : 0 )};
```

Comment:
I think the issue with the spacing here was because lastStepActive wasn't being evaluated correctly. Now that you fix it, the 89px works good and means there's no gap when you scroll to the bottom:

![image](https://user-images.githubusercontent.com/6981253/76581949-62eb6280-64ab-11ea-9758-f14468ce189f.png)

Reply:
89px only works if the button text stays on one line. In the case of paying with credits, or longer-word internationalization, it can cover up the text again:

<img width="296" alt="Screen Shot 2020-03-12 at 9 53 09 PM" src="https://user-images.githubusercontent.com/942359/76582253-1a807480-64ac-11ea-93e2-a92c21e5c282.png">

Reply:
oh good catch! never mind me then :D